yb-voyager/cmd/analyzeSchema.go

```diff
@@ -1213,10 +1149,13 @@ func analyzeSchema() {
 		utils.ErrExit("failed to get the migration status record: %s", err)
 	}
 	analyzeSchemaInternal(msr.SourceDBConf, true)
-	err = checkRedundantIndexes()
-	if err != nil {
-		utils.ErrExit("failed to get the redundant index issues from assessment report: %v", err)
-	}
+
+	//Not populate any perfomance optimizations in analyze-schema report when it is generated with  the command.
```

Comment:
This seems like a slightly round-about way of doing it. 
What i understand is: analyzeSchemaInternal can get you certain performance optimizations
1. in assess-migration, we want the performance optimizations. 
2. in analyze-schema, we don't want the performance optimizations. 

How about instead: analyzeSchemaInternal can have a flag `detectPerformanceOptimzations` and the caller can decide what they want.

Reply:
yeah makes sense

Reply:
Ah good catch on the invalid counts! 
Seems like a good example of how taking shortcut approaches can sometimes lead to bugs 😄

Reply:
Yeah!